yt/data_objects/static_output.py

```diff
@@ -221,7 +233,17 @@ def __init__(
         filename: str,
         dataset_type: Optional[str] = None,
         units_override: Optional[Dict[str, str]] = None,
-        unit_system: str = "cgs",
+        # valid unit_system values include all keys from unyt.unit_systems.unit_systems_registry + "code"
+        unit_system: Literal[
```

Comment:
I don't want to balloon this PR.  But: would it be worthwhile to eventually have this somewhere else, so that we can reference it in other places that we might use it?  For instance, in the various `load`s?

Reply:
yes, that would make sense